packages/vapor/scss/tables/table.scss

```diff
@@ -206,6 +206,10 @@
         td {
             border-bottom: none !important;
         }
+
+        .blankslate-row td.p0 {
+            padding: 0 !important;
```

Comment:
not sure I'm okay with `!important` :smile:

Reply:
:see_no_evil: